### ISSUE TITLE
Drop PHP requirement

### ISSUE THREAD

**Issue by michaelarnauts** [cgarwood/hacontrolpoint#5]
Since you can whitelist IP addresses in HA so they don't have to login, you can drop the PHP proxy in between. This makes it more secure, since the proxy is reachable by everyone, and the API itself only from the device that is running the frontend. An added bonus is a small speed increase because you can directly connect to Home Assistant.

I have this in my `configuration.yaml` of HA:

```yaml
  trusted_networks:
    - 192.168.1.0/24
  cors_allowed_origins:
    - 'null'
    - '*'
```

The `cors_allowed_origins` is needed because Chrome doesn't like AJAX calls when the index.html is opened locally (it might be that this is only during testing when the index.html is opened in a browser).

**Comment by michaelarnauts** [comment]
If home-assistant is configured correctly, this commit doesn't need php: https://github.com/michaelarnauts/hacontrolpoint/commit/4f4e46d334b0d8f8d3e40a364bc429bb8caecd1e

**Comment by michaelarnauts** [comment]
Fixed in af18ec7f43e0367f7fc6efb1d5ed20550d524ba1
